wasm-fiddle-server/server.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express')
```

Comment:
I think for importing it might be better to use ES6 style for javascript

`import express from "express"` or `import { express } from "express"`

It is not as big a deal with this but in general using that style will help with memory allocation as you can limit what you want to import instead of importing files whole. ES6 style is also a lot more modern, I think for the past few years it has become a lot more common in web dev